homework-submission/class16-week1/src/components/Todo.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+class Todo extends React.Component {
+  render() {
+    const { description, deadline } = this.props;
+    return (
+      <li className="todo_item">
+        <h2>
+          Description{" "}
+          <span className="todo_desc">
+            <em>{description}</em>
+          </span>
+        </h2>
+        <h3>
+          Deadline{" "}
+          <span className="todo_deadline">
+            <em>{deadline}</em>
```

Comment:
Instead of using `<em>` you can make the same effect with CSS by `font-style: italic` [check this example](https://www.w3schools.com/cssref/tryit.asp?filename=trycss_font-style)

Reply:
This has been changed as well